tests/t/lib/ProFTPD/Tests/Modules/mod_tls.pm

```diff
@@ -491,7 +496,10 @@ sub list_tests {
     }
   }
 
-  return testsuite_get_runnable_tests($TESTS);
+#  return testsuite_get_runnable_tests($TESTS);
+  return qw(
+    tls_login_with_sni_port_zero_multi_ports_issue932
+  );
```

Comment:
Hrm.  Incomplete attempts for local reproduction; I'll have to clean these up before merging this PR.